pkg/testcase/selinux.go

```diff
@@ -12,56 +11,36 @@ import (
 	"github.com/rancher/distros-test-framework/shared"
 )
 
-var (
-	cmdPrefix  = "sudo ls -laZ"
-	ignoreDir  = "-I .. -I ."
-	rke2       = "/var/lib/rancher/rke2"
-	k3s        = "/var/lib/rancher/k3s"
-	systemD    = "/etc/systemd/system"
-	usrBin     = "/usr/bin"
-	usrLocal   = "/usr/local/bin"
-	grepFilter = "| grep -v \"/\""
-)
-
-const (
-	ctxUnitFile = "system_u:object_r:container_unit_file_t:s0"
-	ctxExec     = "system_u:object_r:container_runtime_exec_t:s0"
-	ctxVarLib   = "system_u:object_r:container_var_lib_t:s0"
-	ctxFile     = "system_u:object_r:container_file_t:s0"
-	ctxConfig   = "system_u:object_r:container_config_t:s0"
-	ctxShare    = "system_u:object_r:container_share_t:s0"
-	ctxRoFile   = "system_u:object_r:container_ro_file_t:s0"
-	ctxLog      = "system_u:object_r:container_log_t:s0"
-	ctxRunTmpfs = "system_u:object_r:container_var_run_t:s0"
-	ctxTmpfs    = "system_u:object_r:container_runtime_tmpfs_t:s0"
-	ctxTLS      = "system_u:object_r:rke2_tls_t:s0"
-	ctxLock     = "system_u:object_r:k3s_lock_t:s0"
-	ctxData     = "system_u:object_r:k3s_data_t:s0"
-	ctxRoot     = "system_u:object_r:k3s_root_t:s0"
-	ctxNone     = "<<none>>"
-	ctxRke2TLS  = "system_u:object_r:rke2_tls_t:s0"
-)
-
 type cmdCtx map[string]string
 
 type configuration struct {
 	distroName string
 	cmdCtx
 }
 
-// TestSelinuxEnabled Validates that containerd is running with selinux enabled in the config
-func TestSelinuxEnabled() {
+// setHelper returns the cluster and product
+func setHelper() (*factory.Cluster, string, error) {
```

Comment:
This func name is confusing. Can we rename to reflect its purpose like `getClusterProduct`

Reply:
sure